Fix:

fileiter.cc:156:3: error: use of undeclared identifier 'abort'
sf-get.cc:51:17: error: use of undeclared identifier 'getenv'
sf-get2.cc:435:7: error: use of undeclared identifier 'getenv'
sf-get3.cc:111:29: error: use of undeclared identifier 'atoi'
--- src/fileiter.cc.orig	2019-05-07 06:40:13.000000000 -0500
+++ src/fileiter.cc	2024-05-06 17:48:43.000000000 -0500
@@ -27,6 +27,7 @@
 #include <config.h>
 
 #include <cstdio>
+#include <cstdlib>
 #include <cstring>
 #include <string>
 
--- src/sf-get.cc.orig	2019-05-07 06:40:13.000000000 -0500
+++ src/sf-get.cc	2024-05-06 17:27:10.000000000 -0500
@@ -28,6 +28,7 @@
 
 #include <config.h>
 
+#include <cstdlib>
 #include <string>
 using std::string;
 
--- src/sf-get2.cc.orig	2019-05-07 06:40:13.000000000 -0500
+++ src/sf-get2.cc	2024-05-06 18:18:55.000000000 -0500
@@ -28,6 +28,7 @@
 
 #include <config.h>
 
+#include <cstdlib>
 #include <string>
 #include <ctype.h>
 
--- src/sf-get3.cc.orig	2019-05-07 06:40:13.000000000 -0500
+++ src/sf-get3.cc	2024-05-06 17:27:35.000000000 -0500
@@ -28,6 +28,8 @@
 
 #include <config.h>
 
+#include <cstdlib>
+
 #include "cssc.h"
 #include "sccsfile.h"
 #include "seqstate.h"
